_includes/partners/x-col.html

```diff
@@ -6,7 +6,7 @@
         <div class="item tablet:grid-col">
           {% if item.image and item.image != '' %}
             <img
-              alt=""
+              alt="{% if item.alt != '' %}{{ item.alt }}{% endif %}"
```

Comment:
Testing this locally, it doesn't seem like it errors in the way that I was hoping for. 😕  On the upside, at least we now have the _option_ to add alt text.